core/services/vrf/integration_test.go

```diff
@@ -0,0 +1,98 @@
+package vrf_test
+
+import (
+	"context"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/solidity_vrf_coordinator_interface"
+
+	"github.com/onsi/gomega"
+	"github.com/smartcontractkit/chainlink/core/services/bulletprooftxmanager"
+	"github.com/smartcontractkit/chainlink/core/services/pipeline"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/guregu/null.v4"
+
+	uuid "github.com/satori/go.uuid"
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/core/services/signatures/secp256k1"
+	"github.com/smartcontractkit/chainlink/core/services/vrf"
+	"github.com/smartcontractkit/chainlink/core/testdata/testspecs"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIntegration_VRFV2(t *testing.T) {
+	config, _, cleanupDB := cltest.BootstrapThrowawayORM(t, "vrf_v2", true)
```

Comment:
Do you need to use the ThrowawayORM?

Reply:
Yeah - this integration test is relying on pg notifs for job spawning and eth txes

Reply:
The pg notifications _should_ work within a transaction, I'm still confused why it doesnt